### PR TITLE
Update downloader.rs for Disable SSL checks

### PR DESCRIPTION
**Pull request by Like0x**

### CODE REVIEW

File: src/downloader.rs [hunk 1]
```diff
@@ -64,6 +64,7 @@ impl Downloader {
 
         Downloader {
             client: reqwest::blocking::ClientBuilder::new()
+                .danger_accept_invalid_certs(true)
```
Comment (CohenArthur) [inline src/downloader.rs:67]: Maybe this should be behind a command line option instead?

Comment (Skallwar) [reply src/downloader.rs:67]: Yeah I agree

**Review by Skallwar — CHANGES_REQUESTED** [review]
Thanks for the PR (and sorry for the delay). I like the feature but I think it should not be enabled by default. Could you add a command line argument for enabling it?

**Comment by Skallwar** [comment]
Closed if favor of #203 which adds CLI arguments. I kept your commit inside the new PR

**Pull request closed without merging**